README.md

```diff
@@ -48,7 +48,7 @@ The CLI can be run with [jbang](https://www.jbang.dev/download/).
 curl -Ls https://sh.jbang.dev | bash -s - app setup
 
 #Install Jlama CLI (will ask if you trust the source)
-jbang app install -j 21 --name=jlama --force https://raw.githubusercontent.com/tjake/Jlama/main/jlama.java
```

Comment:
No need to state Java 21 here as script now requires it. You can still add it if you want to try with higher Java version. 

Also name is redundant as it would derive it from the file name anyway.